src/labels/CustomLabels.labels

```diff
@@ -1913,6 +1913,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>bdiAccountCustomIdError</shortDescription>
         <value>The Account Custom Unique ID setting field {0} must have matching fields in the NPSP Data Import object whose API Names are {1} and {2}.</value>
     </labels>
+    <labels>
+        <fullName>bdiAdditionalObjPredNotFound</fullName>
+        <categories>Data Importer</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>bdiAdditionalObjPredNotFound</shortDescription>
+        <value>Error: Predecessor object not found so record could not be created.</value>
```

Comment:
Is it possible to call out the object dependency field (field label, not field API name)?

Error: record not created. This record is dependent on another record which couldn't be found. The field with this object dependency is [dependency field].

Reply:
After reviewing with Charles, decided to go with the following error message:

Error: record not created. This record is dependent on another record which couldn't be found.